modules/vault/main.tf

```diff
@@ -11,7 +11,7 @@ locals {
 resource "aws_iam_user" "jenkins-x-vault" {
   count = !var.external_vault && var.vault_user == "" && var.use_vault ? 1 : 0
 
-  name = "jenkins-x-vault"
+  name = "jenkins-x-vault-${var.cluster_name}"
```

Comment:
What happens for existing users, who upgrade the module version? Would be good to test that behavior and add a note in the readme here and the eks-jx template repo.

Reply:
probably best to remove this and put it in a separate PR, I can give this a test over the weekend. Dont want to hold up the other changes in this PR. wdyt?